scio-core/src/main/scala/com/spotify/scio/coders/instances/JavaCoders.scala

```diff
@@ -103,7 +103,10 @@ trait JavaCoders extends JavaBeanCoders {
     )
 
   implicit def coderJEnum[E <: java.lang.Enum[E]: ClassTag]: Coder[E] =
-    Coder.beam(SerializableCoder.of(ScioUtil.classOf[E]))
+    Coder.xmap(Coder[String])(
```

Comment:
This *should* be strictly the same as `SerializableCoder` since `Enum` are always serialized as a string ([source](https://docs.oracle.com/javase/6/docs/platform/serialization/spec/serial-arch.html#6469)).
Can we not just use a wrapper around `SerializableCoder` that forces `verifyDeterministic` to return true ? 
It would give use the guarantee that the serialization does not change between Scio versions.

Reply:
we can but I'm not sure if there's a strong reason to? maybe if we had more cases like this?

Reply:
Talked with @jto and this is probably fine for now